src/Controls/src/Core/Shell/Shell.cs

```diff
@@ -94,77 +121,215 @@ static void OnFlyoutItemIsVisibleChanged(BindableObject bindable, object oldValu
+
+		/// <summary>
+		/// Sets the <see cref = "DataTemplate" /> applied to each <see cref = "FlyoutItem" /> object managed by Shell.
+		/// </summary>
+		/// <param name="obj">The object that sets the <see cref = "DataTemplate" /> applied to Item objects.</param>
+		/// <param name="itemTemplate">The <see cref = "DataTemplate" /> applied to Item objects.</param>
 		public static void SetItemTemplate(BindableObject obj, DataTemplate itemTemplate) => obj.SetValue(ItemTemplateProperty, itemTemplate);
 
 		/// <include file="../../../docs/Microsoft.Maui.Controls/Shell.xml" path="//Member[@MemberName='GetBackButtonBehavior']/Docs/*" />
 		public static BackButtonBehavior GetBackButtonBehavior(BindableObject obj) => (BackButtonBehavior)obj.GetValue(BackButtonBehaviorProperty);
-		/// <include file="../../../docs/Microsoft.Maui.Controls/Shell.xml" path="//Member[@MemberName='SetBackButtonBehavior']/Docs/*" />
+
+		/// <summary>
+		/// Sets the behavior of the back button.
+		/// </summary>
+		/// <param name="obj">The object that sets the behavior of the back button.</param>
+		/// <param name="behavior">The back button behavior.</param>
 		public static void SetBackButtonBehavior(BindableObject obj, BackButtonBehavior behavior) => obj.SetValue(BackButtonBehaviorProperty, behavior);
 
-		/// <include file="../../../docs/Microsoft.Maui.Controls/Shell.xml" path="//Member[@MemberName='GetPresentationMode']/Docs/*" />
+		/// <summary>
+		/// Gets the navigation animation that occurs when a page is navigated to with the <see cref = "GoToAsync(ShellNavigationState, bool)" /> method.
+		/// </summary>
+		/// <param name="obj">The object that modifies the tabs visibility.</param>
+		/// <returns>The navigation animation that occurs when a page is navigated to.</returns>
 		public static PresentationMode GetPresentationMode(BindableObject obj) => (PresentationMode)obj.GetValue(PresentationModeProperty);
-		/// <include file="../../../docs/Microsoft.Maui.Controls/Shell.xml" path="//Member[@MemberName='SetPresentationMode']/Docs/*" />
+
+		/// <summary>
+		/// Sets the navigation animation that plays when a <see cref="Page"/> is navigated to with the <see cref = "GoToAsync(ShellNavigationState, bool)" /> method.
+		/// </summary>
+		/// <param name="obj">The object that modifies the tabs visibility.</param>
+		/// <param name="presentationMode">Defines the navigation animation that occurs when a page is navigated.</param>
 		public static void SetPresentationMode(BindableObject obj, PresentationMode presentationMode) => obj.SetValue(PresentationModeProperty, presentationMode);
 
-		/// <include file="../../../docs/Microsoft.Maui.Controls/Shell.xml" path="//Member[@MemberName='GetFlyoutBehavior']/Docs/*" />
+		/// <summary>
+		/// Gets the navigation animation that occurs when a page is navigated to with the <see cref = "GoToAsync(ShellNavigationState, bool)" /> method.
```

Comment:
This is the same as the GetPresentationMode docs. Are they the same? 

This seems to get a page's flyout behavior, so should be similar to what I suggested for SetBackButtonBehavior (but as a getter)

Reply:
@jsuarezruiz  ?

Reply:
Oops, fixed.